### ISSUE TITLE
Feature Suggestion: math preview work with a shortcut

### ISSUE THREAD

**Issue by diego898** [SublimeText/LaTeXTools#1017]
I think it would be great to have a fourth option to `preview_math_mode`, which is something like `manual`, meaning I specify some shortcut to toggle the preview on/off.

**Comment by r-stein** [comment]
Related #914.
You mean, that you press a shortcut inside a math environment and it opens?

Because you can create a shortcut to change the settings to disable or enable it for the view.

**Comment by diego898** [comment]
I see - so it would be a shortcut to turn it on/off

**Comment by r-stein** [comment]
Creating a shortcut to turn it on/off would be very trivial. Adding a shortcut to open a phantom would not so trivial, because the current behavior works with listeners not with command triggers.

**Comment by diego898** [comment]
Got it!

**Comment by ViktorQvarfordt** [comment]
@r-stein How do I create such a shortcut most easily?

**Comment by r-stein** [comment]
You can create a keybinding to change it for the view:

``` js
    // disable preview math mode
    {
        "keys": ["ctrl+l", "ctrl+t", "ctrl+m"],
        "command": "set_setting",
        "args": {
            "setting": "preview_math_mode",
            "value": "none"
        }
    },
    // enable preview math mode for the selected math env
    {
        "keys": ["ctrl+l", "ctrl+t", "ctrl+shift+m"],
        "command": "set_setting",
        "args": {
            "setting": "preview_math_mode",
            "value": "selected"
        }
    },
```

**Comment by ViktorQvarfordt** [comment]
Cool, thanks! Do you know if it's possible to have a key bind that toggles between non-boolean settings? (I.e. not two different key binds, but just one.)

**Comment by r-stein** [comment]
Yes it is possible, but it requires a small context listener, because the default setting context does not seem to work with more than booleans.
